build.gradle

```diff
@@ -59,23 +71,33 @@ artifacts {
     archives javadocJar
 }
 
-test {
-    useTestNG()
+tasks.withType(Test) {
+    useTestNG {
+        preserveOrder = true
+    }
 
-    // Log the results on every run.
-    testLogging {
-        outputs.upToDateWhen {false}
-        showExceptions = true
-        exceptionFormat = 'full'
-        showStackTraces = true
-        showStandardStreams = true
+    testlogger {
+        showStandardStreams true
+        theme 'mocha'
     }
+}
+
+integration {
```

Comment:
Are there docs/source where I can see where you got this from?

Reply:
This task is defined up on line 21 using the TestSets plugin. I was debating adding some comments to clarify the structure of the whole build file. I think your question just convinced me it would be worthwhile.